scalableminds/webknossos
# Add nuclei inferral UI

This PR adds a rather simple UI to start a nuclei inferral job for a dataset

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- setup a local wk-worker
- open a dataset with a single color layer in view mode or a tracing. There go to the info tab and use the "Permium Features" Dropdown to open the nuclei inferral modal. In the modal, the start button should directly be available, as there is only one color layer. Starting the job should work.
- Now do the same for a dataset with multiple color layers. This time the modal should have a select for the user to choose the color layer that should be used for the nuclei inferral.

### Issues:
- fixes  #5631

------
